CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.2)
+
+project(todocli LANGUAGES CXX)
+
+include_directories("${PROJECT_SOURCE_DIR}/lib/")
```

Comment:
it's a bit nitpicking for your project as it won't be an error but it should be CMAKE_SOURCE_DIR.

https://stackoverflow.com/a/32030551 for the difference